doc/api.rst

```diff
@@ -179,7 +179,8 @@ For example::
   struct fmt::formatter<point> : nested_formatter<double> {
     auto format(point p, format_context& ctx) const {
       return write_padded(ctx, [=](auto out) {
-        return format_to(out, "({}, {})", nested(p.x), nested(p.y));
+        return format_to(out, "({}, {})", this->nested(p.x),
```

Comment:
Does it make sense to mention that specifying `this` here is only necessary when using GCC <7?

Reply:
I don't think this makes sense.